### PR TITLE
[OSDOCS-9186]: Include a note on DDOS protection Service for OSD on Google Cloud in OSD service definition doc

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-9186
Link to docs preview:

https://69717--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/osd-service-definition#network-security_osd-service-definition
QE review:
- [X] QE has approved this change.

Note QE  approved original ticket https://issues.redhat.com/browse/OSDOCS-8885. This update is a duplicate of that ticket. 
Additional information:

image
**Hello reviewers As an FYI, this PR is based on a request from the PM to duplicate the info regarding "Network Security" found here: https://69717--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/policy-process-security#network-security_policy-process-security for consistency purposes. This info was reviewed/approved by QE for that ticket, and this ticket change has also been approved by SME per request of the PM**

### CODE REVIEW

**Comment by jaybeeunix** [comment]
The change looks good to me.

File: modules/sdpolicy-security.adoc [hunk 1]
```diff
@@ -77,7 +77,8 @@ $ oc adm policy add-cluster-role-to-group self-provisioner system:authenticated:
 
 [id="network-security_{context}"]
 == Network security
-With {product-title} on AWS, AWS provides a standard DDoS protection on all Load Balancers, called AWS Shield. This provides 95% protection against most commonly used level 3 and 4 attacks on all the public facing Load Balancers used for {product-title}. A 10-second timeout is added for HTTP requests coming to the haproxy router to receive a response or the connection is closed to provide additional protection.
+Each {product-title} cluster is protected by a secure network configuration at the cloud infrastructure level using firewall rules (AWS Security Groups or Google Cloud Compute Engine firewall rules). {product-title} customers on AWS are also protected against DDoS attacks with link:https://docs.aws.amazon.com/waf/latest/developerguide/ddos-overview.html[AWS Shield Standard].
```
Comment (cbippley) [inline modules/sdpolicy-security.adoc:80]: Maybe rephrase to avoid parenthesis. 
```suggestion
Each {product-title} cluster is protected by a secure network configuration at the cloud infrastructure level using firewall rules, either AWS Security Groups or Google Cloud Compute Engine firewall rules.
```

**Review by cbippley — COMMENTED** [review]
LGTM!
